### DISCUSSION TITLE
Does not work with Horsch Pronto 6AS / Maestro 8RC (from modhub - Agrartechnik Nordeifel) ?

### DISCUSSION THREAD

**Discussion by Szmiti** [Mogli12/AIVehicleExtension#263] [Previous version]
(Tool selected throughout is the trailed SW3500S with seed tank - not the mounted rear drills)

Strange behaviour - even with all other mods removed ....

AI Veh Ex ON (ext. helper on in-game option): Does not seem to recognise field boundaries --> AI Veh Ext text flashes yellow-green-yellow at end of field / run.   Circle setting does not seem to work at all - sown ground not recognized - tractor just turns and does not follow sown ground or field boundary.

AI Veh Ex OFF: moves forward but after turning at end of field, runs back over ground already sown - ie. turns "wrong" way.  Also, only seems to work at all starting next to sown ground - if starting in open area does not work - only drives forward slowly and AI Veh Ex text stays yellow - drill not lowered and not tuned on if not already.

(With AI Veh Ext disabled, works OK with vanillia helper only)

**Comment by Mogli12** [comment]
I work on it, but it is a difficult case

**Comment by Mogli12** [comment]
Hi,
Please check version v0.0.6.8. Please note that combinations of cultivator and sowing machine do not work on already cultivated land.
Regards

**Comment by Szmiti** [comment]
Hello,
Thanks for your efforts !  I tested v0.0.6.8. (on harvested / non-cultivated ground) and it is now usable but still with some issues.   There is a (very) big overlap - especially when AI veh. mode set to 'circle' to follow around field boundary - acts like the correct width not detected ?  This can be offset by using the 'width offset' in the menu but this doesn't work either at corners / turning.  With a 90deg corner, the cult-drill combi is lowered with almost the middle on the field boundary and the correct edge alignent is only achieved after driving forwards some distance.  So there is some issue with corners too it seems ?  However, as this is all only because of one mod, if this is very difficult to improve, maybe it's not worth spending a lot of time on this ?  My feeling is that maybe you should spend time elswhere to solve other issues and add maybe new features to this fantastic AIVE mod (which I use all the time) ?!  It's a nice cult-drill combi mod but there _are_ other options and I can live without it ....  Thanks again !
